assets/src/edit-story/components/checklistCard/styles.js

```diff
@@ -70,6 +71,22 @@ export const Cta = styled.div`
   margin: 0;
 `;
 
+export const CheckboxCtaLabel = styled(Text).attrs({
+  forwardedAs: 'label',
+  size: THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL,
+})`
+  color: ${({ theme }) => theme.colors.fg.linkNormal};
+`;
+
+export const CheckboxCtaContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin: 14px 0 6px;
+  label {
+    margin: 0 8px;
+  }
+`;
```

Comment:
kinda figure these and the icon below on line  120 will just move to the videoOptimization check.